go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -731,29 +733,31 @@ func (oc *DefaultNetworkController) getEgressFirewallACLDbIDs(namespace string,
 		})
 }
 
-func getNodeInternalAddrsToString(node *kapi.Node) []string {
-	v4, v6 := util.GetNodeInternalAddrs(node)
-	var addrs []string
-	for _, addr := range []net.IP{v4, v6} {
-		if addr != nil {
-			addrs = append(addrs, addr.String())
-		}
+func getNodeInternalAddrsToString(node *kapi.Node) ([]string, error) {
```

Comment:
should we rename this function? getNodeHostAddrs or something? I'm surprised a function for this doesnt already exist. Maybe it should go into the util pkg?

Reply:
agreed I see this logic duplicated in too many places, let me create a new commit with the new util and use that here and everywhere else

Reply:
this is done.